src/dsp_tools/xmllib/helpers.py

```diff
@@ -0,0 +1,61 @@
+import json
+from typing import Any
+from typing import Iterable
+
+
+def create_label_to_name_list_node_mapping(
+    project_json_path: str,
+    list_name: str,
+    language_label: str,
+) -> dict[str, str]:
+    """
+    Often, data sources contain list values named after the "label" of the JSON project list node, instead of the "name"
+    which is needed for the `dsp-tools xmlupload`.
+    In order to create a correct XML, you need a dictionary that maps the "labels" to their correct "names".
+
+    Alternatively, consider using the method create_json_excel_list_mapping(), which also creates a dictionary, but maps
+    values from your data source to list node names from the JSON project file, based on similarity.
+
+    Args:
+        project_json_path: path to a JSON project file (a.k.a. ontology)
+        list_name: name of a list in the JSON project (works also for nested lists)
```

Comment:
I think the part with the nested lists is unnecessary to mention

```suggestion
        list_name: name of a list in the JSON project
```